src/lib/layout/logs.svelte

```diff
@@ -423,3 +431,39 @@
         </div>
     </section>
 {/if}
+
+<style>
+    :global(.limited-code-height) {
+        max-height: 50vh;
+        overflow: auto !important;
+    }
+
+    .code-panel-compact-header .u-flex {
+        gap: 1.5rem !important;
+    }
+
+    @media (max-width: 768px) {
+        .code-panel-compact-header {
+            row-gap: 1rem;
+            flex-direction: column !important;
+        }
+
+        .code-panel-content {
+            padding: unset !important;
+            padding-block: 1.5rem !important;
+        }
+
+        .mobile-only-inline-20-padding {
+            padding-inline: 1.5rem !important;
+        }
+
+        .mobile-only-block-start-20-padding {
+            padding-block-start: 1.5rem !important;
+        }
+
+        .code-panel-content.grid-1-2 {
+            display: unset !important;
+            row-gap: unset !important;
+        }
```

Comment:
Are all these importants really needed? I think some are due to pink v1 classes, but all of them?

Reply:
Addressed.